app/.babelrc

```diff
@@ -1,25 +1,16 @@
 {
   "presets": [
-    "env",
-    "react"
-  ],
-  "plugins": [
-    "react-hot-loader/babel",
-    "transform-runtime",
-    "transform-decorators-legacy",
-    "transform-class-properties",
-    "transform-object-rest-spread",
+    "@babel/preset-react",
     [
-      "react-intl",
```

Comment:
This plugin has been used for extracting messages from files. Where is it being replaced?